integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,246 @@
+     | maven | io.vertx:vertx-core | 3.6.2 |
+     | maven | io.vertx:vertx-core | 3.6.1 |
+     | maven | io.vertx:vertx-core | 3.6.0 |
+     | maven | org.json:json | 20180813 |
+     | maven | org.json:json | 20180130 |
+     | maven | org.json:json | 20171018 |
+     | maven | org.json:json | 20170516 |
+     | maven | org.python:jython | 2.7.1b3 |
+     | maven | org.python:jython | 2.7.1b2 |
+     | maven | org.clojure:clojure | 1.10.1-beta2 |
+     | maven | org.clojure:clojure | 1.10.1-beta1 |
+     | maven | org.clojure:clojure | 1.10.0 |
+     | pypi | clojure_py | 0.2.4 |
+     | pypi | six | 1.10.0 |
+     | pypi | ansicolors | 1.1.8 |
+     | pypi | flask | 1.0.2 |
+     | pypi | flask | 1.0.1 |
+     | pypi | flask | 1.0 |
+     | pypi | numpy | 1.16.3 |
+     | pypi | scipy | 1.2.1 |
+     | pypi | scipy | 1.2.0 |
+     | pypi | pygame | 1.9.6rc1 |
+     | pypi | pygame | 1.9.5 |
+     | pypi | pyglet | 1.4.0a1 |
+     | pypi | pyglet | 1.3.2 |
+     | pypi | requests | 2.21.0 |
+     | pypi | requests | 2.20.1 |
+     | pypi | requests | 2.20.0 |
+     | pypi | dash | 1.0.0a1 |
+     | pypi | dash | 0.43.0rc3 |
+     | pypi | dash | 0.43.0rc2 |
+     | pypi | dash | 0.43.0rc1 |
+     | pypi | pudb | 2018.1 |
+     | pypi | pudb | 2017.1.4 |
+     | pypi | pudb | 2017.1.3 |
+     | pypi | pudb | 2017.1.2 |
+     | pypi | pudb | 2017.1.1 |
+     | pypi | pytest | 3.2.1 |
+     | pypi | pytest | 3.2.2 |
+ 
+     
+     Scenario: Check the component analysis V2 REST API endpoint for unknown ecosystem
+        Given System is running
+          Given Three scale preview service is running
+          
+          When I acquire the use_key for 3scale
+          Then I should get the proper user_key
+          
+
+          When I start v2 component analyses really_unknown_ecosystem/foobar/1.0.0 with user_key
+          Then I should get 400 status code
+          And I should receive a valid JSON response
+
+   
+  Scenario: Check the component analysis REST V2 API endpoint for unknown component in NPM ecosystem
+    Given System is running
+      Given Three scale preview service is running
+  
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+    
+     When I start v2 component analyses npm/really_unknown_component/1.0.0 with user_key
+      Then I should get 202 status code
+      And I should receive a valid JSON response
+
+  
+  Scenario: Check the component analysis V2 REST API endpoint for unknown component in PyPi ecosystem
+    Given System is running
+     Given Three scale preview service is running
+      
+      When I acquire the use_key for 3scale
+      Then I should get the proper user_key
+      
+
+     When I start v2 component analyses pypi/really_unknown_component/1.0.0 with user_key
+     Then I should get 202 status code
+      And I should receive a valid JSON response
+      
+    
+  Scenario: Check if the snyk login link works
```

Comment:
What is the snyk login link here? What if something breaks on snyk side, do we start failing our e2e?

Reply:
i was checking if the link is valid by sending a request and checking if it gives 200 . i can remove is it isnt valid :smile:

Reply:
not really sure if we need this in our e2e. Its not related to any of our code. why should we be evaluating it

Reply:
okay will remove this